administration/src/cards/pdf/PdfElements.ts

```diff
@@ -0,0 +1,13 @@
+export type Coordinates = {
+  x: number
+  y: number
+}
+
+export function mmToPt(mm: number) {
+  return (mm / 25.4) * 72
+}
+
+export type PdfElementRenderer<T extends Record<string, any>> = (info: T) => void
+export type PdfElement<P extends Record<string, any>, T extends Record<string, any>> = (
+  options: P
+) => PdfElementRenderer<T>
```

Comment:
I am wondering whether it would be more appealing to just specify the `PdfElement` options (`P`) in the project configs instead of `PdfElement(options: P)`.
I think we could then make the API a bit less confusing / more explicit:

```suggestion
export type PdfElementRenderer<ConfigOptions extends Record<string, any>, DynamicOptions extends Record<string, any>> = (
  configProps: ConfigOptions , dynamicProps: DynamicOptions
) => void
```

Then, the `PdfQrCodeElement.ts` would become something like:
```ts
export type PdfQrCodeConfigOptions = {
  size: number
} & Coordinates

type QrCodeDynamicOptions = {
  page: PDFPage
  qrCode: PdfQrCode
}

const PdfQrCodeRenderer: PdfElementRenderer<PdfQrCodeConfigOptions, QrCodeDynamicOptions> = ({ size, x, y }, { page, qrCode }) => {
  const qrCodeSizePdf = mmToPt(size)
  const qrCodeXPdf = mmToPt(x)
  const qrCodeYPdf = page.getSize().height - qrCodeSizePdf - mmToPt(y)

  const qrCodeContent = new QrCode({
    qrCode: qrCode,
  }).toBinary()

  drawQRCode(qrCodeContent, qrCodeXPdf, qrCodeYPdf, qrCodeSizePdf, page, false)
}
```